pkg/discovery/processor/cluster_processor.go

```diff
@@ -9,7 +9,13 @@ import (
 	"github.com/turbonomic/kubeturbo/pkg/discovery/util"
 	"github.com/turbonomic/kubeturbo/pkg/kubeclient"
 	"k8s.io/client-go/pkg/api/v1"
-	"strings"
+	"time"
+)
+
+const (
+	workers = 10
+	sleepInterval = 10 * time.Second
+	totalWaitTime = 60 * time.Second
```

Comment:
`totalWaitTime` should be configurable?

Reply:
Good question. Not sure.

Reply:
Let's make it configurable, we can parse it as an argument when running kubeturbo. 60 can be the default value.